### PR TITLE
build: add codeowners file

### LINKED ISSUE #13: Missing CODEOWNERS
@bcoe @sofisl can you please add an appropriate CODEOWNERS

I suspect yoshi-nodejs should be top level code owner but we'll need to include the product team for samples at a minimum. 

@carlinyuen is the product lead

### PR DESCRIPTION
**Pull request by sofisl**
Fixes #13

### CODE REVIEW

**Comment by bcoe** [comment]
@sofisl we would need to create a corresponding team here I believe:

https://github.com/orgs/googleapis/teams

@JustinBeckwith how have you been approaching this: I think we'd create an `area120` team, do we still create the team in `sloth` as well?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+# The yoshi-nodejs team is the default owner for nodejs repositories.
+*     @googleapis/yoshi-nodejs @googleapis/area120-tables
```
Comment (bcoe) [inline .github/CODEOWNERS:9]: I think we'd probably make the team `area120`, vs., a team for the specific package.

**Comment by sofisl** [comment]
@bcoe, I think they are already a team, they are just not externally visible: https://github.com/googleapis/nodejs-area120-tables/pull/16

I'm not sure how this affects the codeownership honestly. But they do exist!

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
> @bcoe, I think they are already a team, they are just not externally visible: #16

What's slightly weird to me, is I'd expect an `area120` team, not an `area120-tables`, but I'm okay with following Python's lead for now.

**Pull request merged**